generators/server/templates/pom.xml.ejs

```diff
@@ -45,6 +45,18 @@
             </releases>
         </repository>
         <%_ } _%>
+        <%_ if (ENTANDO_BUNDLE_BOM_VERSION.indexOf('-PR-') > -1) { _%>
```

Comment:
I would rather add that in the JS code creating a specific variable for that purpose to make it more understandable. What this statement is supposed to be?

Reply:
Variable added